packages/renderer/src/lib/kube/details/KubeContainerPorts.svelte

```diff
@@ -0,0 +1,27 @@
+<script lang="ts">
+import type { V1ContainerPort } from '@kubernetes/client-node';
+
+import Cell from '/@/lib/details/DetailsCell.svelte';
+
+interface Props {
+  ports?: V1ContainerPort[];
+}
+
+let { ports }: Props = $props();
+</script>
+
+{#if ports && ports.length > 0}
```

Comment:
`ports?.length > 0`?

Reply:
Error: 'ports.length' is possibly 'undefined'. (ts)

{#if ports?.length > 0}

:(